testsuite/run_forge.sh

```diff
@@ -184,8 +185,8 @@ if [ "$FORGE_RUNNER_MODE" = "local" ]; then
     # more file descriptors for heavy txn generation
     ulimit -n 1048576
 
-    cargo run -p forge-cli -- --suite $FORGE_TEST_SUITE --mempool-backlog 1500 --avg-tps $LOCAL_TPS_THRESHOLD \
-        --max-latency-ms $LOCAL_P99_LATENCY_MS_THRESHOLD \
+    cargo run -p forge-cli -- --suite $FORGE_TEST_SUITE --workers-per-ac 10 --avg-tps $FORGE_RUNNER_TPS_THRESHOLD \
+        --max-latency-ms $LOCAL_P99_LATENCY_MS_THRESHOLD --duration-secs $FORGE_RUNNER_DURATION_SECS  \
```

Comment:
nit: should we also get rid of `LOCAL_P99_LATENCY_MS_THRESHOLD` in favor of the more generic e.g. `FORGE_RUNNER_P99_LATENCY_MS_THRESHOLD` instead?

Reply:
Yes, we can do that as a follow up - the latency metrics is something I will need to work on revamping. Currently we are measuring p99 e2e latency, which is noisy and also not sufficient.